Extensions/Video/videoruntimeobject.js

```diff
@@ -311,10 +342,9 @@ gdjs.VideoRuntimeObject.gdjsCallbackRuntimeScenePaused = function(
 /**
  * When a scene is resumed, resume any video previously paused.
  * @private
+ * @param {gdjs.RuntimeScene} runtimeScene The {@link gdjs.RuntimeScene} this objects has to be removed from.
```

Comment:
The {@link gdjs.RuntimeScene} that is being resumed

Reply:
fixed